schema/generic.json

```diff
@@ -198,17 +198,11 @@
             ],
             "additionalProperties": false
         },
-        "Address": {
+        "CommonAddressComponents": {
             "type": "object",
-            "description": "Physical address information, typically used as part of billing or shipping address objects.",
+            "description": "Address components common to all address combinations.",
             "pii": true,
             "properties": {
-                "line1": {
-                    "type": "string",
-                    "description": "Street name with house number.",
-                    "minLength": 1,
-                    "example": "501 Twin Peaks Blv"
-                },
                 "line2": {
```

Comment:
Why `line2` not removed?

Reply:
`line2` is still required by all address types, it contains non-standard information such as `unit G.03` or `floor 3, apartment 31`